components/NavBar.js

```diff
@@ -1,5 +1,71 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import styles from '../styles/Navbar.module.scss'
+
+import { useAppContext } from '../context/app-context'
+
 export default function NavBar() {
+  const { sharedState } = useAppContext()
+
+  const courses = sharedState.courses.map((course, i) => {
+    const modules = course.Modules.map((mod, j) => {
+      return (
+        <Link key={j} href='/moduleDashboard'>
+          <a>{mod.name}</a>
+        </Link>
+      )
+    })
+
     return (
-        <div></div>
+      <div key={i} className={styles.course}>
+        <Link href='/courseDashboard'>
+          <a>{course.Name}</a>
+        </Link>
+        <div className={styles.content}>
+          <Link href='/moduleDashboard'>
+            <a>All Modules</a>
+          </Link>
+
+          {modules}
+
+          <Link href='/addModuleForm'>
+            <a>+ Add New Module</a>
+          </Link>
+        </div>
+      </div>
     )
-}
\ No newline at end of file
+  })
+
+  return (
+    <nav className={styles.navbar}>
+      <section className={styles.container}>
+        <div className={styles.logo}>
+          <Image src='/icon.png' alt='Course chart icon' height={50} width={50} />
+        </div>
+        <h1>CourseChart</h1>
+      </section>
+
+      <section className={styles.courseButtons}>
+        <Link href='/'>
+          <a className={styles.course}>All Course Dashboard</a>
+        </Link>
+
+        {courses}
+
+        <Link href='/addCourseForm'>
+        <a className={styles.addCourse}>+ Add New Course</a>
+        </Link>
+      </section>
+
+      <section className={styles.navButtons}>
+        <Link href='/instructions'>
+          <a>Instructions</a>
+        </Link>
+
+        <Link href='/instructions'>
```

Comment:
Temporarily. We didn't create a page for the About section so I didn't have anywhere to link to. I'm going to leave it for now in case we decide to add an About page.